db/warehouse/migrate/20241112181349_set_missing_timestamps.rb

```diff
@@ -0,0 +1,39 @@
+class SetMissingTimestamps < ActiveRecord::Migration[7.0]
+  def up
+    # Destination Clients
+    import_batch = []
```

Comment:
This should be defined inside the `find_in_batches` loop? With the current setup it looks like clients will be re-imported repeatedly with each batch.